packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/Choices.js

```diff
@@ -93,37 +95,46 @@ export function Choices({
 
   return (
     <StyledGrid
+      role='menu'
       rowsCount={rowsCount}
     >
       {filteredChoiceIds.map((choiceId, index) => {
         const choice = task.choices?.[choiceId] || {}
         const selected = selectedChoiceIds.indexOf(choiceId) > -1
         const src = task.images?.[choice.images?.[0]] || ''
-        const hasFocus = autoFocus && (index === focusIndex)
-        const tabIndex = (index === focusIndex) ? 0 : -1
+        const hasFocus = index === focusIndex
+        let tabIndex = -1
+        if (focusIndex === null && index === 0) {
+          tabIndex = 0
+        } else if (index === focusIndex) {
+          tabIndex = 0
+        }
 
         return (
-          <ChoiceButton
+          <li
             key={choiceId}
-            choiceId={choiceId}
-            choiceLabel={task.strings.get(`choices.${choiceId}.label`)}
-            disabled={disabled}
-            hasFocus={hasFocus}
-            onChoose={onChoose}
-            onKeyDown={handleKeyDown}
-            selected={selected}
-            src={src}
-            tabIndex={tabIndex}
-            thumbnailSize={thumbnailSize}
-          />
+            role='menuitem'
```

Comment:
```suggestion
            role='menuitemcheckbox'
            aria-checked={ selected ? ‘true’ : ‘false’ }
```

When you add a role, you should also check which states apply to that role, and add them if necessary. 

A checkbox might be good here because many items can be selected at a time.

Reply:
After reading the MDN docs for checkbox menu items, I’m wondering if this role and state should be set on the button. 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitemcheckbox_role

Reply:
Yeah, the HTML in the example here has the `menuitem` roles on the clickable links. That was my mistake. Menu items should be interactive elements like links, buttons and checkboxes. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role

Reply:
Great suggestions, thank you!
Per the linked ARIA docs I added `role='presentation'` to the `li`s.